mappings/net/minecraft/client/search/IdentifierSearchableContainer.mapping

```diff
@@ -5,4 +5,7 @@ CLASS dwr net/minecraft/client/search/IdentifierSearchableContainer
 	FIELD c identifierFinder Ljava/util/function/Function;
 	FIELD d entries Ljava/util/List;
 	FIELD e entryIds Lit/unimi/dsi/fastutil/objects/Object2IntMap;
+	METHOD a compare (Ljava/lang/Object;Ljava/lang/Object;)I
+		ARG 1 object1
+		ARG 2 object2
```

Comment:
Just curious, how does this actually compare objects?

Reply:
```java
    protected int compare(T object1, T object2) {
        return Integer.compare(this.entryIds.getInt(object1), this.entryIds.getInt(object2));
    }
```